skymp5-server/cpp/server_guest_lib/script_classes/PapyrusLeveledItem.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+#include "IPapyrusClass.h"
+
+class PapyrusLeveledItem final : public IPapyrusClass<PapyrusLeveledItem>
```

Comment:
this class has to be constructed but I don't remember where the factory is. it's somewhere near

Reply:
Found and added.